server/mailet/mock-smtp-server/src/main/java/org/apache/james/mock/smtp/server/MockSmtpBehaviors.java

```diff
@@ -22,6 +22,8 @@
 import java.util.List;
 import java.util.Objects;
 
+import org.apache.james.mock.smtp.server.model.MockSMTPBehavior;
```

Comment:
Is that an unused import?

Reply:
no, that class was located int model package